reading-data/reading-data-assign.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+import numpy as np
+
+cars = pd.read_csv("mtcars.csv")
+beer = pd.read_fwf("beer.txt")
+nhl = pd.read_excel("NHL 2015-16.xlsx")
+
+cars.to_csv("new_mtcars.csv")
+beer.to_csv("new_beer.txt")
+nhl.to_excel("new nhl 2015-2016.xlsx")
```

Comment:
Should also read these new files back and compare them to the original dataframe object (cars, beer, etc.). The nhl data doesn't need to be an exact match but everything else should be.